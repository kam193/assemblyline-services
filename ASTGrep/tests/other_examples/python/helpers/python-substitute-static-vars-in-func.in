def some_func():
    a = 17
    call_sth(a)

    if sth > 19:
       b = 32
    c = b

    d = 87
    if sth > 19:
        x, d = cl(), 32
    e = d