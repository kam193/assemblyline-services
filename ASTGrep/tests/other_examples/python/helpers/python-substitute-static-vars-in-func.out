def some_func():
    a = 17
    call_sth(17)

    if sth > 19:
       b = 32
    c = b

    d = 87
    if sth > 19:
        x, d = cl(), 32
    e = d